apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -32,3 +32,25 @@ body {
 	white-space: nowrap;
 	border: 0;
 }
+
+html {
+	scrollbar-width: thin;
+	scrollbar-color: var(--color-border-primary) var(--color-background-secondary);
+}
+
+html::-webkit-scrollbar {
+	width: 8px;
+	height: 8px;
+}
+
```

Comment:
By the way, don't we need to remove scrollbar styles from textarea? please check that new styles apply to it and to tables

Reply:
Oh, yeah, I thought we'll leave them for the textarea. New styles apply to tables, and textarea - confirmed 😅.